Lib/test/support/_interpreters.py

```diff
@@ -0,0 +1,235 @@
+"""Subinterpreters High Level Module."""
+
+import logging
+import _xxsubinterpreters as _interpreters
+
+__all__ = ['Interpreter', 'SendChannel', 'RecvChannel',
+            'is_shareable', 'create_channel',
+            'list_all_channels', 'get_current',
+            'get_main', 'create']
```

Comment:
There are some missing:

```suggestion
__all__ = ['Interpreter', 'SendChannel', 'RecvChannel',
            'is_shareable', 'create_channel',
            'list_all_channels', 'get_current',
            'get_main', 'create', 'list_all', 'RunFailedError']
```

The channel-related exception types should be added too.

FWIW, I sometimes find it helpful to group the related names:

```suggestion
__all__ = [
    'Interpreter', 'get_current', 'get_main', 'create', 'list_all',
    'SendChannel', 'RecvChannel',
    'create_channel', 'list_all_channels', 'is_shareable',
    'RunFailedError',
    ]
```